New post for astro view transitions

kaslee opened the pull request:
New file and author entries added

Changes proposed in this pull request:

- New post for astro view transitions
- Author entries added

addyosmani commented:
Reviewed internally by @rachelandrew (with suggested fixes merged in this version and a note to be mindful of usage of "we").

addyosmani reviewed (approved):
We did an offline review with fixes landing in a separate commit. LGTM

rachelandrew commented on site/en/blog/astro-view-transitions/index.md:
```diff
@@ -0,0 +1,164 @@
+layout: 'layouts/blog-post.njk'
+title: Astro View Transitions
+description: 
+  Highlight how the Astro community embraced View Transitions when Chrome was shaping up the API and the journey towards landing first-class support for it in Astro + persistent islands
```
```suggestion
  Highlight how the Astro community embraced View Transitions when Chrome was shaping up the API and the journey towards landing first-class support for it in Astro + persistent islands.
```

rachelandrew commented on site/en/blog/astro-view-transitions/index.md:
````diff
@@ -0,0 +1,164 @@
+
+One of the earliest attempts involved a full-page “smart transition” that would automatically compare every new page of HTML to the current HTML and then attempt to replace every element at once using JavaScript. It was clever, but error-prone.
+
+```javascript
````
all the code is indented, can you remove the white space at the beginning of lines?

addyosmani replied:
Ack. we will do this shortly.

kaslee replied:
Can you check now.

rachelandrew commented on site/en/blog/astro-view-transitions/index.md:
````diff
@@ -0,0 +1,164 @@
+    })
+```
+
+Thankfully, the Astro community showed that the View Transitions API was already powerful without the product over-complicating things. After seeing some early demos, the Astro team were sold on the idea of a simple Astro API that could match the browser APIs as closely as possible. Instead of seeing View Transitions as an invisible implementation detail hidden inside of Astro, they could be dire ...
````
```suggestion
Thankfully, the Astro community showed that the View Transitions API was already powerful without the product over-complicating things. After seeing some early demos, the Astro team were sold on the idea of a simple Astro API that could match the browser APIs as closely as possible. Instead of seeing View Transitions as an invisible implementation detail hidden inside of Astro, they could be directly exposed to the developer. Aligning Astro directly with the web platform and reducing overall complexity on the frontend.
```

rachelandrew reviewed (changes requested):
Please see my note about whitespace in code examples.

rachelandrew reviewed (approved):
Thank you, you should be able to merge this when ready but if you have trouble let me know.

kaslee commented:
@rachelandrew The latest commit addresses feedback from the Astro team.

Also, I do not have write access to the repo and cannot merge. Request your help for this.
